selection_tools/run_prepare_histograms.sub

```diff
@@ -1,7 +1,7 @@
 Universe              = vanilla
 executable            = run_prepare_histograms.sh
 #			process			part	    input_user (dust space with input histograms)
-arguments             = tta_mAlp-10GeV 	$(ProcId)   jniedzie
+arguments             = tta_mAlp-10GeV 	$(ProcId)   jniedzie 0
```

Comment:
Can you add a comment on the meaning of this "0"? Btw. if this is just to know if we should reduce number of histograms, then in the C++ file I'd change the name from "ctau_option" to "reduce_histograms".